src/components/sequencing/SequenceEditor.svelte

```diff
@@ -221,16 +233,56 @@
       showFailureToast('Error copying sequence.json to clipboard');
     }
   }
+
+  async function copySeqNClipboard() {
+    try {
+      await navigator.clipboard.writeText(editorSequenceView.state.doc.toString());
+      showSuccessToast('SeqN copied to clipboard');
+    } catch {
+      showFailureToast('Error copying SeqN to clipboard');
+    }
+  }
+
+  function toggleSeqJsonEditor() {
+    toggleSeqJsonPreview = !toggleSeqJsonPreview;
+  }
 </script>
 
 <CssGrid bind:columns={commandFormBuilderGrid} minHeight={'0'}>
-  <CssGrid bind:rows={$userSequencesRows} minHeight={'0'}>
+  <CssGrid rows={toggleSeqJsonPreview ? $userSequencesRows : $userSequencesJSONCollapsedRows} minHeight={'0'}>
```

Comment:
If these aren't meant to write back to the store, then we can do away with the store and just initialize with a string representation of the layout within the component.

Reply:
What is the purpose of writing them back to the storage? I was a little confused when I saw this in the code.

Reply:
it will be removed